pandas/core/common.py

```diff
@@ -2,23 +2,22 @@
 Misc tools for implementing data structures
 """
 
-import re
 import collections
 import numbers
+import re
 from datetime import datetime, timedelta
 from functools import partial
 
-import numpy as np
-import pandas as pd
 import pandas.algos as algos
 import pandas.lib as lib
 import pandas.tslib as tslib
+
+import pandas as pd
 from pandas import compat
-from pandas.compat import (long, zip, map, string_types,
-                           iteritems)
```

Comment:
put things back in this file as these are not substantive changes (and things getting changed elsewhere)